### DISCUSSION TITLE
Maybe the throughput between TGI and vLLM should be updated

### DISCUSSION THREAD

**Discussion by zhaoyang-star** [vllm-project/vllm#478] [General]
I just notice that TGI has support Paged Attention by integrating vLLM in TGI. The [PR#516](https://github.com/huggingface/text-generation-inference/pull/516) has merged and be available in TGI v0.9.2.

**Comment by zhuohan123** [comment]
Hi! We are testing the performance of TGI and will update the performance results afterward. You can track the progress at #381
